assets/info.plist

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleIdentifier</key>
```

Comment:
Reduce indent

Reply:
it's a tap, so I assume you mean two spaces instead?

Reply:
made it two spaces...

Reply:
> it's a tap, so I assume you mean two spaces instead?

Just thought the tags within the `dict` tag should be consistent. Probably better if they are all tabs (or 4 spaces).

Reply:
looks like I was a little faster 😅 Since we have two spaces everywhere for indentation, I assumed that would be right here as well

Reply:
That works! Thanks!